code/game/atoms_movable.dm

```diff
@@ -430,7 +428,7 @@
 			A.Entered(src, old_loc)
 
 			for(var/atom/movable/AM in loc)
-				AM.Crossed(src,no_tp)
+				AM.Crossed(src)
```

Comment:
dropping an arg here?

Reply:
The arg is fake. Doesn't conform to the proc's format and doesn't do anything AFAIK